Makefile

```diff
@@ -229,3 +229,7 @@ coverage/report-xml: coverage/combine
 .PHONY: coverage/report
 coverage/report-html: coverage/combine
 	docker-compose $(COMPOSE_OPTS) run --rm -T backend bash -c "cd /single-cell-data-portal && coverage html --data-file=$(COVERAGE_DATA_FILE) -i --skip-empty"
+
+.PHONY: promote-staging-to-prod # set GITHUB_PAT env var OR call as "make GITHUB_PAT=<github_pat> promote-staging-to-prod"
+promote-staging-to-prod:
+	curl -X POST https://api.github.com/repos/chanzuckerberg/single-cell-data-portal/dispatches -H 'Accept: application/vnd.github.everest-preview+json' --data '{"event_type": "deployment"}' --header 'authorization: Bearer $(GITHUB_PAT)'
```

Comment:
So using the api is a bit annoying because it requires a token, whereas simply using a `git` command, a la promoting a branch and then pushing, does not (uses user's local ssh creds). I would tend to favor the git cli approach instead for this reason.